File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] temp = s.split(" ");
        int min = Integer.parseInt(temp[0]);
        int max = Integer.parseInt(temp[0]);
        for (String str : temp) {
            int num = Integer.parseInt(str);
            if (min > num) {
                min = num;
            }
            if (max < num) {
                max = num;
            }
        }
        answer = Integer.toString(min) + " " + Integer.toString(max);
        return answer;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    answer = ''
    list = s.split(" ")
    for i in range(len(list)):
        list[i] = int(list[i])
    list.sort()
    answer = str(list[0])
    answer += " "
    answer += str(list[len(list) -1])
    return answer